engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/WarningsTest.java

```diff
@@ -57,27 +52,28 @@ public static void disposeContext() {
 
   @Test
   public void doubleWithWarningsWrap() {
-    var warn1 = Warning.create(ensoContext, "w1", this);
-    var warn2 = Warning.create(ensoContext, "w2", this);
-    var value = 42L;
-
-    var with1 = WithWarnings.wrap(ensoContext, value, warn1);
-    var with2 = WithWarnings.wrap(ensoContext, with1, warn2);
-
-    assertEquals(value, with1.getValue());
-    assertEquals(value, with2.getValue());
-    Assert.assertArrayEquals(
-        new Object[] {warn1}, with1.getWarningsArray(WarningsLibrary.getUncached(), false));
-    Assert.assertArrayEquals(
-        new Object[] {warn1, warn2}, with2.getWarningsArray(WarningsLibrary.getUncached(), false));
+    ContextUtils.executeInContext(ctx, () -> {
+      var warn1 = Warning.create(ensoContext, "w1", this);
+      var warn2 = Warning.create(ensoContext, "w2", this);
+      var value = 42L;
+
+      var with1 = AppendWarningNodeGen.getUncached().execute(null, value, warn1);
```

Comment:
You may want to put `getUncached()` into `AppendWarningNode`. That would allow us to _not reference_ the generated class in the code other than `AppendWarningNode` itself.

Reply:
Done in 82f8506e7ba28723b80f1e5b46dc4644c69aa6af